en/frames_vtol/README.md

```diff
@@ -0,0 +1,56 @@
+# VTOL Airframes
+
+PX4 supports virtually all VTOL configurations:
+
+  * Tailsitters (duo and quadrotors in X and plus configuration)
+  * Tiltrotors (Firefly Y6)
+  * QuadPlane VTOL (standard plane plus quad)
+
+The VTOL codebase is the same codebase as for all other airframes and just adds additional control logic, in particular for transitions.
+
+<!-- 
+> **Note** All these VTOL configurations have been actively test-flown and are ready to use. Ensure that there is airspeed sensor attached to the system as it is used by the autopilot when its safe to perform the transition.
+-->
+
+This section contains build logs and instructions for assembling and configuring a number of VTOL vehicle frames.
+
+
+## Key Configuration Parameters
```

Comment:
@sanderux Hi Sander, Do you think you might be able to comment on the above point?